projects/cashmere/src/lib/tabs/tab-set/tab-set.component.ts

```diff
@@ -38,7 +38,7 @@ export function invalidDefaultTab(tabVal: string) {
 export class TabSetComponent implements AfterContentInit {
     _routerEnabled: boolean = false;
     private _direction: string = 'vertical';
-    private _defaultTab: string = '0';
+    private _defaultTab: string | number = '0';
```

Comment:
Since we're accepting a number, shouldn't the default just be `0`? That way we can save < 1ms of parsing that every single time.